src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -0,0 +1,35 @@
+.pf-c-notification-badge {
+  --pf-c-notification-badge--after--Top: 0;
+  --pf-c-notification-badge--after--Right: 0;
+  --pf-c-notification-badge--after--Width: var(--pf-global--spacer--sm);
+  --pf-c-notification-badge--after--Height: var(--pf-global--spacer--sm);
+  --pf-c-notification-badge--after--BackgroundColor: var(--pf-global--active-color--400);
+  --pf-c-notification-badge--after--BorderRadius: var(--pf-global--BorderRadius--lg);
+
+  // Modifiers
+  --pf-c-notification-badge--m-read--after--BackgroundColor: transparent;
+  --pf-c-notification-badge--m-unread--after--BackgroundColor: var(--pf-global--active-color--400);
+
+  position: relative;
+
+  &::after {
+    position: absolute;
+    top: var(--pf-c-notification-badge--after--Top);
+    right: var(--pf-c-notification-badge--after--Right);
+    display: inline-block;
```

Comment:
you shouldn't need this. setting `position: absolute;` will make it a block element.